packages/shared/src/components/auth/RegistrationForm.tsx

```diff
@@ -141,6 +152,8 @@ export const RegistrationForm = ({
 
   const isNameValid = !hints?.['traits.name'] && isSubmitted;
   const isUsernameValid = !hints?.['traits.username'] && isSubmitted;
+  const isExperienceLevelValid =
+    !isSubmitted || !hints?.['traits.experienceLevel'];
```

Comment:
If the form has not been submitted, the isExperienceLevelValid would always be false. I'm not sure how the other fields handle that, but this was causing the experience level warning to be shown as red (screenshot below).

<img width="417" alt="Screenshot 2024-04-24 at 08 34 24" src="https://github.com/dailydotdev/apps/assets/9974711/64a37091-37e9-45c5-bf19-5062e951b3b8">

If you think about it, this is actually correct logic. E.g. if the form is not submitted, we do not mark the field as invalid.

Reply:
> if the form is not submitted, we do not mark the field as invalid.

Agree with that, just because the others were handled differently it caught my eye. Still don't understand 100% how come they are handled differently.